src/components/call-to-action/_macro.njk

```diff
@@ -0,0 +1,20 @@
+{% macro onsCallToAction(params) %}
+    {% from "components/button/_macro.njk" import onsButton %}
+    <div class="call-to-action">
+        <div class="container">
+            <div class="grid grid--flex grid--vertical-center grid--no-wrap@s">
+                <div class="grid__col col-auto u-flex-shrink@s">
+                    <p class="call-to-action__desc u-fs-r u-mb-no@s"><strong>{{ params.headingText }}</strong> {{ params.paragraphText }}</p>
```

Comment:
After our visit to DAC yesterday I have a feeling`<strong>` tags cause screen readers to pause. I'll check this on JAWS, NVDA, and VoiceOver today

Reply:
Yes we saw that too @bameyrick. Maybe test if it does the same with `<em>`

Reply:
JAWS and voiceover reads out tag by tag. The string in the strong tag could be a heading which would be announced in the same way. Perhaps it should be a heading so screen readers would read it out if all headings are read out?

Reply:
Not sure if matters if screen reader pauses in this instance.

Reply:
If it is a heading and a description, wouldn't it make more sense for this to be a heading tag followed by a paragraph tag then?

Reply:
Yes that's what I'm saying. I'll update to be a `h2` and `p`